test/ffmpeg-qsv/encode/encoder.py

```diff
@@ -36,10 +36,7 @@ def gen_output_opts(self):
       else:
         opts += " -q {qp}"
     if vars(self).get("quality", None) is not None:
-      if self.codec in ["jpeg",]:
-        opts += " -global_quality {quality}"
-      else:
-        opts += " -preset {quality}"
+      opts += " -global_quality {quality}"
```

Comment:
The `preset` and `global_quality` options have different meanings in ffmpeg-qsv.  This will change the behavior of existing tests.  In the context of vaapi-fits, we interpret the"quality" parameter in the encode tests as the "preset" (i.e. compression level 1 thru 7) for all middleware encoders (except for jpeg codec). 

The problem comes from the fact that every middleware uses a different name for setting up the "compression level".  FFmpeg-qsv uses `preset`, ffmpeg-vaapi uses `quality`, gst-msdk uses `target-usage` and gst-vaapi uses `quality-level` to set the "compression level".  So, at vaapi-fits level, it was necessary to normalize the the name to a single meaning (i.e. "quality").  Perhaps "tu" (i.e. target-usage) would have been a better normalized name choice to avoid confusion.  Maybe it's time to change that!  Changing it is cumbersome, though, since it would require writing a test name translation for the baselines and updating all user test configs.  The other problem is we don't have control over other peoples test configs and it would mess them up... although, I'm not sure if others are using this test framework, yet.  Ideally, we should add some sort of versioning scheme for test config and baseline files so that we can provide automatic translation support.

 It's a mess and confusing, I know!  But for now, "quality" **must** be interpreted as the compression level (except for jpeg codec) in vaapi-fits across all middleware tests for proper consistency.

Reply:
So, can you show me how to change the code here ?

Reply:
Maybe you can introduce a different parameter to influence global_quality setting for ICQ tests.

Reply:
FWIW, ICQQuality is the field name in the msdk mfx structure.  In ffmpeg-qsv, `mfx.ICQQuality = avctx->global_quality`.  In gst-msdk, `param->ICQQuality = thiz->qpi`.  AFAICT, valid range is [1, 51].

So how about "icq_quality" as the vaapi-fits normalized param name which can be translated to the appropriate user param at the middleware test level.

Reply:
PR updated, please review.